corehq/apps/smsbillables/tasks.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from datetime import date
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.conf import settings
+
+from corehq.apps.smsbillables.models import (
+    SmsBillable,
+    SmsGatewayFeeCriteria,
+)
+from corehq.util.log import send_HTML_email
+from dimagi.utils.dates import add_months_to_date
+
+
+@periodic_task(run_every=crontab(day_of_month='1', hour=5, minute=0), queue='background_queue', acks_late=True)
+def send_gateway_fee_report_out():
+    backend_api_ids = SmsGatewayFeeCriteria.objects.values('backend_api_id').distinct()
+    first_day_previous_month = add_months_to_date(date.today(), -1)
+    billables_in_month = SmsBillable.objects.filter(
+        date_sent__year=first_day_previous_month.year,
+        date_sent__month=first_day_previous_month.month,
+    )
```

Comment:
this should probably also filter only for [global ](https://github.com/dimagi/commcare-hq/blob/7f0a177a022fda8f430930829ce89db70a8952a1/corehq/apps/sms/models.py#L1982) backends so we avoid domain-owned ones.

does it also need to filter on is_valid?

Reply:
Added a filter on is_valid.

The SMSBillables for domain-owned backends don't record a gateway fee, so they won't count towards the sum of gateway charges.